src/fr/articles/listbox/index.njk

```diff
@@ -3,24 +3,25 @@ title: "Les listbox"
 abstract: "Exemple de liste ARIA."
 titleBeforeTag: true
 date: "2018-01-07"
+updateDate: "2025-02-28"
 tags:
   - component
 ---
 
 <h2>Introduction</h2>
 
 <p><abbr>ARIA</abbr> met à disposition un grand nombre de composants qui peuvent servir dans les interfaces riches (onglet, accordéon…).
-Nous allons étudier le cas des <i lang="en">listbox</i>. Il s'agit d'un composant qui permet à l’utilisateur de sélectionner un ou plusieurs éléments à partir d’une liste de choix.</p>
+Nous allons étudier le cas des <i lang="en">listbox</i> ou zone de liste. Il s'agit d'un composant qui permet à l’utilisateur de sélectionner un ou plusieurs éléments à partir d’une liste de choix.</p>
 <p>Les éléments d’une <i lang="en">listbox</i> sont statiques, une <i lang="en">listbox</i> n’est pas destinée à recevoir des éléments interactifs (tels que des liens, des boutons…), en revanche celle-ci peut contenir des images à l’inverse d’un <code>select</code>.</p>
-<p>Plus d’infos&nbsp;: <a href="https://w3c.github.io/aria-practices/#Listbox" hreflang="en" lang="en"><abbr>WAI</abbr>-<abbr>ARIA</abbr> Authoring Practices</a>.</p>
+<p>Plus d’infos&nbsp;: <a href="https://www.w3.org/WAI/ARIA/apg/patterns/listbox/" hreflang="en" lang="en"><abbr>WAI</abbr>-<abbr>ARIA</abbr> Authoring Practices Guide</a>.</p>
```

Comment:
Not sure about this link, it may be a bit confusing:
- visible name: 'WAI-ARIA Authoring Practices Guide'
- page destination: Listbox Pattern